app/src/androidTest/java/net/osmtracker/Layouts/MockData.java

```diff
@@ -0,0 +1,14 @@
+package net.osmtracker.Layouts;
```

Comment:
What do you think about changing the package name to `net.osmtracker.data` and  class name to `Mocks`?

Reply:
I agree that's a better structured way to sort the classes. I'll change it